_posts/2020-07-27-solidity-v0.7.0-release-announcement.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.7.0 Release Announcement'
+date: '2020-07-27'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity 0.7.0](https://github.com/ethereum/solidity/releases/tag/v0.7.0) is a major breaking release of the Solidity compiler and language.
+
+This release does not include many features but rather changes that require a backwards-incompatible change in syntax or semantics. For an explanation of the features, please see the [documentation](https://solidity.readthedocs.io/en/latest/070-breaking-changes.html).
+
+Most notably, a cleanup of visibility and state mutability has been performed and several unpopular keywords have been removed. Furthermore, types with mappings in memory are now disallowed and a better clarification of types with regards to shift and exponentiation has been implemented.
```

Comment:
I will try to find a better wording for `better clarification of types with regards to shift and exponentiation`